test/k8s-integration/cluster.go

```diff
@@ -292,28 +292,42 @@ func clusterUpGKE(gceZone, gceRegion string, numNodes int, imageType string, use
 			return fmt.Errorf("failed to create a new service: %v", err)
 		}
 
-		// TODO: change to read CLOUDSDK_API_ENDPOINT_OVERRIDES_CONTAINER env var
-		service.BasePath = "https://test-container.sandbox.googleapis.com/" // e.g test, staging env URL
+		service.BasePath = os.Getenv("CLOUDSDK_API_ENDPOINT_OVERRIDES_CONTAINER")
```

Comment:
note: for prod, the env variable will be unset. All though we only have test jobs for test and staging env. If we want to run jobs for prod, this maynot work.

Reply:
oh I see... what's the endpoint for prod? I think a if statement here would be fine

Reply:
it must be "container.sandbox.googleapis.com" i think. can you check in your setup?